lib/ts/nextjs.ts

```diff
@@ -39,6 +39,7 @@ export default class NextJS {
         response: any
     ): Promise<T> {
         return new Promise<T>(async (resolve: any, reject: any) => {
+            request.fromNextJS = true;
```

Comment:
You should add a test that if this wrapper is used, then the request has this variable.